### DISCUSSION TITLE
Assignment of Public IP To Management interface in Azure

### DISCUSSION THREAD

**Discussion by szym3n** [Security-Onion-Solutions/securityonion#9019] [General]
Hi SO Experts,

I'm trying to utilize one SO Heavy Node deployed in the cloud as a syslog receiver for some syslog events from another tool.
The other tool is Cloud-native, so I figured if I assign a public IP to the existing Management Interface on the existing SO Heavy Node in addition to its private one, I could then send syslog events to that IP and receive+analyze them from within SO.

Is this possible?
We created an ACL in Azure that allows only specific IPs to send syslog, and everything seems good from the Azure side, however the Security Onion server doesn't appear to be sensing any traffic directed at the Public IP.

I have used so-allow and hostgroups to add a few firewall rules allowing some specific, known public IP addresses to send to port 514 to that Azure Server, and I'm still not seeing ANYTHING going inbound towards that IP address.
Not only that, the IFCONFIG doesn't seem to recognize that the VM has a public IP AND a private IP assigned to its first NIC.

I read that re-assignment of IP addresses in a Distributed Environment like ours is not really possible, and I'd have to re-deploy, but before I do that, I wanted to know whether what I'm trying to do is even possible.

Do you have any advice on how I could set this up?

Thanks,
Szymon

**Comment by TotieBash** [comment]
This is not the answer but on my SO build instead of using "ifconfig" I use "ip a" .  The ifconfig does not show my second ip address as compared to "ip a" but I didn't investigate further on how to make it work with ifconfig.  That is all, wait for the heavy hitters to really answer your questions.

ifconfig
![image](https://user-images.githubusercontent.com/86748904/199026769-8bb68a79-1211-45a9-bd5d-a060972e52d6.png)


ip a
![image](https://user-images.githubusercontent.com/86748904/199027387-2fbd0c46-71ff-4386-8ff4-694210a05c6b.png)

**Comment by szym3n** [comment]
Hi Totie,

Yeah, you're right the public IP is visible.
That said, for whatever reason I am still not seeing any traffic on it and am unable to send syslog that way.

I'm still wondering what the best way to do this would be.

Regards,
Szymon

**Reply by TotieBash** [reply]
To troubleshoot syslog issue like this I normally use tcpdump and logger utility.
1. I use logger utility to send test syslog message from a different linux machine to SO.  From a different linux machine:
logger -n <SO-MGMT-IP-address> -P 514 "TEST MESSAGE"
ie. logger -n 912.168.10.10 -P 514 "TEST MESSAGE"

2. I use tcpdump on SecurityOnion to see if those syslog message are reaching the wire.  From the SO shell:
tcpdump -i <SO-MGMT-INT> port 514 -nnv 
ie. tcpdump -i eth1 port 514 -nnv 

Good luck..
